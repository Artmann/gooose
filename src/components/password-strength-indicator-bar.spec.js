import PasswordStrengthIndicatorBar from './password-strength-indicator-bar';
import React from 'react';
import renderer from 'react-test-renderer';

describe('PasswordStrengthIndicatorBar', () => {
  it('renders correctly when deactivated', () => {
    const tree = renderer
      .create(<PasswordStrengthIndicatorBar color='#330000' />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('renders correctly when activated', () => {
    const tree = renderer
      .create(<PasswordStrengthIndicatorBar color='#330000' active={true} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
