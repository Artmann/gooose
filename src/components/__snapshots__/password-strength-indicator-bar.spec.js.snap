// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PasswordStrengthIndicatorBar renders correctly when activated 1`] = `
<div
  className="sc-bdVaJa dwnnPn"
  color="#330000"
/>
`;

exports[`PasswordStrengthIndicatorBar renders correctly when deactivated 1`] = `
<div
  className="sc-bdVaJa dTPqtA"
  color="transparent"
/>
`;
